DCScribe/Core/Scribe.cs

```diff
@@ -54,25 +54,29 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
                 var scribeToken = scribeTokenSource.Token;
 
                 // Clear the database as we start from scratch each time around
-                await _databaseClient.ClearTableAsync();
-
-                /*
-                 * A queue containing all the unit updates to be processed. We populate
-                 * this queue in a separate thread to make sure that slowdowns in unit
-                 * processing do not impact the rate at which we can receive unit updates
-                 *
-                 * We clear the queue each time we connect
-                 */
-                var queue = new ConcurrentQueue<Unit>();
-                _rpcClient.UpdateQueue = queue;
-
-                var tasks = new[]
-                {
-                    _rpcClient.StreamUnitsAsync(scribeToken), // Get the events and put them into the queue
-                    ProcessUnitQueue(queue, scribeToken), // Process the queue events into the units dictionary
-                    ProcessAirbaseUpdates(scribeToken), // Process Airbase updates by calling an APi on a timer
-                    ProcessMarkPanelUpdates(scribeToken) // Process MarkPanel updates by calling an API on a long timer
-                };
+                await _databaseClient.ClearTableAsync();                
+
+                var tasks = new List<Task>();
+                
+                if (GameServer.Tasks.ProcessAirbaseUpdates.Enabled) {
+                    tasks.Add(ProcessAirbaseUpdates(scribeToken)); // Process Airbase updates by calling an APi on a timer
+                }
+                if (GameServer.Tasks.ProcessMarkPanelUpdates.Enabled) {
+                    tasks.Add(ProcessMarkPanelUpdates(scribeToken)); // Process MarkPanel updates by calling an API on a long timer
+                }
+                if (GameServer.Tasks.RecordUnitPositions.Enabled) {
+                        /*
+                        * A queue containing all the unit updates to be processed. We populate
+                        * this queue in a separate thread to make sure that slowdowns in unit
+                        * processing do not impact the rate at which we can receive unit updates
+                        *
+                        * We clear the queue each time we connect
+                        */
+                        var queue = new ConcurrentQueue<Unit>();
+                        _rpcClient.UpdateQueue = queue;
+                        tasks.Add(_rpcClient.StreamUnitsAsync(GameServer.Tasks.RecordUnitPositions.PollRate, scribeToken)); // Get the events and put them into the queue
```

Comment:
```suggestion
                        var pollRate = GameServer.Tasks.RecordUnitPositions.PollRate;
                        tasks.Add(_rpcClient.StreamUnitsAsync(pollRate, scribeToken)); // Get the events and put them into the queue
```

Just to keep line length down.